### DISCUSSION TITLE
Feature request: HasConflict trigger

### DISCUSSION THREAD

**Discussion by ssfxate** [Resnovas/smartcloud#218] [Ideas]
Hey, thank you for the great GH action. I really happy with it.

Is it possible to label a PR due to occured conflict with the base branch?

**Comment by TGTGamer** [comment]
Hey,

Thank for the great suggestion. For some reason I'm having issues accessing the GitHub API on my mobile, so I cannot confirm this is possible right now. 

In theory if I there's an endpoint for when GitHub detects a conflict, I can simply access this within the conditions, but I don't know if this would be sufficient for your desires? 

The other one is to develop a tool for checking all the open PR's and then testing for conflicts? 

I want to develop a tool to look at all changes made to a PR for technical debt checking, maybe this could be combined?

**Reply by TGTGamer** [reply]
So looking through the API documentation, the simple method I was hoping would work doesn't have an endpoint that I can use. What I would need to do is compare the commits on all the pull requests and test for a conflict that way. 

This information has been documented here: https://stackoverflow.com/questions/34011782/how-to-resolve-conflict-with-merging-with-github-api

I will add it to the backlog to do, however, I have other higher priority issues to address first, which will enable much-needed functionality in this action. 

I will aim to have this in progress by the end of the month, once other git actions have been enabled within the action.

**Reply by ssfxate** [reply]
I know about the feature you are talking about. When we see a potential conflict with other opened PRs.
But for my personal purpose it'd be enought to label a PR just due to this condition.
![image](https://user-images.githubusercontent.com/18144023/133074188-23b0148e-3d43-4363-9750-a30e04ac24c1.png)

**Reply by ssfxate** [reply]
Potential conflict checker is like this https://github.com/marketplace/actions/potential-conflicts-checker.
But It's overhead I think for the first approach.

**Comment by TGTGamer** [comment]
New issue created: #219
